### PR TITLE
Correct index for blockcommit.top_inactive test

### PR DESCRIPTION
**Pull request by Yingshun**
It's caused by new feature - add index for the active
layer in disk chain.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by Yingshun** [comment]
**Test result:**
**Before fix:**
` (1/1) type_specific.io-github-autotest-libvirt.virsh.blockcommit.normal_test.single_chain.file_disk.local.relative_path.notimeout.shallow.top_inactive: FAIL: error: invalid argument: commit of 'vda' active layer requires active flag\n (26.91 s)`

**After fix:** 
_libvirt 6.0_
```
(.libvirt-ci-venv-ci-runtest-kRwBa6) [root@dell-per730-66 ~]# avocado run --vt-type libvirt --vt-machine-type q35 virsh.blockcommit.normal_test.single_chain.file_disk.local.relative_path.notimeout.shallow.top_inactive --vt-connect-uri qemu:///system
JOB ID     : cb569deba5abe3fd1d7d259fcc9dce9569df7e27
JOB LOG    : /root/avocado/job-results/job-2020-04-14T22.10-cb569de/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blockcommit.normal_test.single_chain.file_disk.local.relative_path.notimeout.shallow.top_inactive: PASS (28.28 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 29.76 s
(.libvirt-ci-venv-ci-runtest-kRwBa6) [root@dell-per730-66 ~]# 
```

_libvirt 5.6_
```
(.libvirt-ci-venv-ci-runtest-7ctyz5) [root@yicui-host1 ~]#  avocado run --vt-type libvirt --vt-machine-type q35 virsh.blockcommit.normal_test.single_chain.file_disk.local.relative_path.notimeout.shallow.top_inactive --vt-connect-uri qemu:///system
JOB ID     : 682982f4612a3798620023ef3732e1b6276bdcaf
JOB LOG    : /root/avocado/job-results/job-2020-04-14T22.10-682982f/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blockcommit.normal_test.single_chain.file_disk.local.relative_path.notimeout.shallow.top_inactive: PASS (31.21 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 32.62 s
(.libvirt-ci-venv-ci-runtest-7ctyz5) [root@yicui-host1 ~]# 

```

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py [hunk 1]
```diff
@@ -452,14 +452,37 @@ def check_chain_backing_files(disk_src_file, expect_backing_file=False):
...
+                        disk_xml = disk
+                        break
+
+                top_index = 1
+                try:
+                    top_index = disk_xml.backingstore.index
+                except AttributeError:
+                    pass
+                else:
+                    top_index = int(top_index)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py:469]: I assume this line code is moved to  position just after line 465 and else statement is useless

Comment (Yingshun) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py:469]: No, it will do the operation in else if there is no exception.

Comment (smitterl) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blockcommit.py:469]: Wow, I didn't know that existed :D - https://geek-university.com/python/the-try-except-else-statements/

**Pull request merged**